VIS/Areas/VIS/Scripts/app/Framework/email.js

```diff
@@ -6,6 +6,9 @@
         //local variables          
         this.frame = null;
         this.windowNo = VIS.Env.getWindowNo();
+        /** Tab panel-adding cc & bcc emails ** Dt: 28/06/2021 ** Modified By: Kumar **/
+        this.ccmail = '';
+        this.bccmail = '';
```

Comment:
Don't create public variable and don't assign values to them directly from other class. Create methods to set value of variable in email class and invoke these mothods from your class.